contrib/changelog/prod/3.19.0_1726656718.md

```diff
@@ -0,0 +1,3 @@
+* As quantum computing advances, so do our measures to protect your data. You can now enable post-quantum encryption against the cyber threats of the future. It’s always better to be prepared than to wish you were. Or, as they say: "Be quantum-proof." Use the command: nordvpn set PQ <enable>.
```

Comment:
$ nordvpn set PQ on
Command 'PQ' doesn't exist.

$ nordvpn set pq on
Post-quantum VPN is set to 'enabled' successfully.

Reply:
$ nordvpn set PQ <enable>
bash: syntax error near unexpected token `newline'

Reply:
Misleading recommendation what command and how should be used